### PR TITLE
FISH-490 Remove JAXB API usage from the Payara Eclipse tools

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/internal/SystemLibrariesSetting.java [hunk 1]
```diff
@@ -63,14 +69,18 @@ public void setLibraryList(ArrayList<Library> libraries) {
...
+            }
 
             SystemLibrariesSetting settings = null;
             try (InputStream stream = settingsXmlFile.getContents()) {
-                settings = (SystemLibrariesSetting) JAXBContext.newInstance(SystemLibrariesSetting.class).createUnmarshaller().unmarshal(stream);
+                SAXParserFactory saxParserFactory = SAXParserFactory.newInstance();
+                SAXParser saxParser = saxParserFactory.newSAXParser();
+                SysLibHandler handler = new SysLibHandler();
+                saxParser.parse(stream, handler);
+                settings = handler.getSystemLibrariesSetting();//(SystemLibrariesSetting) JAXBContext.newInstance(SystemLibrariesSetting.class).createUnmarshaller().unmarshal(stream);
```
Comment (dmatej) [inline plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/internal/SystemLibrariesSetting.java:83]: I see a comment to delete :-)

File: plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/internal/SystemLibrariesSetting.java [hunk 2]
```diff
@@ -81,22 +91,40 @@ public static SystemLibrariesSetting load(IProject project) {
...
+                }
+                writer.writeEndElement();
+                writer.writeCharacters(NEW_LINE);
+            }
+            writer.writeEndElement();
+            writer.writeCharacters(NEW_LINE);
+            writer.writeEndDocument();
 
+            writer.flush();
+            writer.close();
```
Comment (dmatej) [inline plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/internal/SystemLibrariesSetting.java:127]: Resource leak on exception - you should close the writer in finally or use trywith (you can simply create a private method XMLStreamWriter createWriter();

**Review by dmatej — CHANGES_REQUESTED** [review]
I didn't try it yet, but except my comments it seems alright

**Review by dmatej — APPROVED** [review]

**Comment by jGauravGupta** [comment]
Thanks @dmatej

**Pull request merged**